easybuild/easyconfigs/g/gemmi/gemmi-0.6.3-GCCcore-11.3.0.eb

```
@@ -0,0 +1,47 @@
+# Thomas Hoffmann, EMBL Heidelberg, structures-it@embl.de, 2023/11
+easyblock = 'CMakePythonPackage'
+
+name = 'gemmi'
+version = '0.6.3'
+
+homepage = 'https://gemmi.readthedocs.io/'
+description = """
+Gemmi is a library, accompanied by a set of programs, developed primarily for  
+use in macromolecular crystallography (MX). For working with:
+
+macromolecular models (content of PDB, PDBx/mmCIF and mmJSON files), refinement
+restraints (CIF files), reflection data (MTZ and mmCIF formats), data on a 3D  
+grid (electron density maps, masks, MRC/CCP4 format) crystallographic symmetry.
+Parts of this library can be useful in structural bioinformatics (for symmetry-
+aware analysis of protein models), and in other molecular-structure sciences   
+that use CIF files (we have the fastest open-source CIF parser).
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+toolchainopts = {'opt': True}
+
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+checksums = ['d7fa5e2a96f2093900a8ceaf835d2761562b5c42403e3f4cc98d32bb94f6a091']
+
+github_account = 'project-gemmi'
+
+runtest = 'cpptest test_disulf test'
+
+configopts = '-DUSE_PYTHON=1 '
+configopts += '-DUSE_FORTRAN=1  '
+configopts += '-DPYTHON_INSTALL_DIR=%(installdir)s/lib/python%(pyshortver)s/site-packages '
+
+builddependencies = [
+    ('pybind11', '2.9.2'),
+    ('CMake', '3.24.3'),
+    ('binutils', '2.38'),
+]
+dependencies = [('Python', '3.10.4')]
```

small nitpick on order of easyconfig parameters:

```suggestion
github_account = 'project-gemmi'
source_urls = [GITHUB_SOURCE]
sources = ['v%(version)s.tar.gz']
checksums = ['d7fa5e2a96f2093900a8ceaf835d2761562b5c42403e3f4cc98d32bb94f6a091']

builddependencies = [
    ('pybind11', '2.9.2'),
    ('CMake', '3.24.3'),
    ('binutils', '2.38'),
]
dependencies = [('Python', '3.10.4')]

configopts = '-DUSE_PYTHON=1 '
configopts += '-DUSE_FORTRAN=1  '
configopts += '-DPYTHON_INSTALL_DIR=%(installdir)s/lib/python%(pyshortver)s/site-packages '

runtest = 'cpptest test_disulf test'
```